tests_utils/date_time_utils.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+class DateTimeUtils:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def current_datetime(format_date="%d/%m/%Y %H:%M"):
+        """Gets the current datetime in the specified format."""
```

Comment:
Can I get you to write out the docstring with any arguments that need to be passed in?

So for this one, it'd be:
```
"""
Gets the current datetime in the specified format.

Args:
    format_date (str): [Optional] The format to return the current datetime in.  Defaults to dd/mm/yyyy hh:mm if not provided.
"""
```

If we do this, it populates IntelliSense when people hover over / go to use the class, so they get a bit more of a helpful description about what they need to provide.